perf_tests/random_index_and_tensor_search.py

```diff
@@ -0,0 +1,77 @@
+from __future__ import annotations
+
+import random
+import os
+
+from locust import events, task, between, run_single_user
+from locust.env import Environment
+from wonderwords import RandomSentence, RandomWord
+import marqo
+
+from common.marqo_locust_http_user import MarqoLocustHttpUser
+
+INDEX_NAME = os.getenv('MARQO_INDEX_NAME', 'locust-test')
+
+
+class IndexingUser(MarqoLocustHttpUser):
+    fixed_count = 1
+    wait_time = between(1, 2)
+    s = RandomSentence()
+
+    @task
+    def add_docs(self):
+        # Generate random documents batch (5-10 docs) with random length description of 1-5 sentences
+        random_docs = [{
+            'title': self.s.sentence(),
+            'description': ' '.join([self.s.sentence() for j in range(random.randint(1, 5))])
+        } for i in range(random.randint(5, 10))]
+
+        self.client.index(INDEX_NAME).add_documents(documents=random_docs, tensor_fields=['title', 'description'])
+
+
+class SearchUser(MarqoLocustHttpUser):
+    wait_time = between(1, 2)
+    w = RandomWord()
+
+    @task
+    def search(self):
```

Comment:
Will a locust spawned user just run this test infinitely until the time runs out?

Reply:
Yes, correct.

Reply:
Thanks!